x/inflation/keeper/inflation.go

```diff
@@ -36,80 +40,97 @@ func (k Keeper) MintCoins(ctx sdk.Context, newCoin sdk.Coin) error {
 	return k.bankKeeper.MintCoins(ctx, types.ModuleName, newCoins)
 }
 
-// DistributeMintedCoins implements distribution of minted coins from mint to external modules.
-func (k Keeper) DistributeMintedCoin(ctx sdk.Context, mintedCoin sdk.Coin) error {
+// AllocateMintedCoins allocates minted coins from the inflation to external
+// modules
+func (k Keeper) AllocateMintedCoin(ctx sdk.Context, mintedCoin sdk.Coin) error {
 	params := k.GetParams(ctx)
-	proportions := params.DistributionProportions
+	proportions := params.InflationDistribution
 
-	// allocate staking incentives into fee collector account to be moved to on next begin blocker by staking module
-	stakingIncentivesCoins := sdk.NewCoins(k.GetProportions(ctx, mintedCoin, proportions.Staking))
-	err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, k.feeCollectorName, stakingIncentivesCoins)
+	// Allocate staking rewards into fee collector account
+	stakingRewards := sdk.NewCoins(k.GetProportions(ctx, mintedCoin, proportions.StakingRewards))
+	err := k.bankKeeper.SendCoinsFromModuleToModule(
+		ctx,
+		types.ModuleName,
+		k.feeCollectorName,
+		stakingRewards,
+	)
 	if err != nil {
 		return err
 	}
 
-	// TODO replace poolIncentives with usageIncentives
-	// // allocate pool allocation ratio to pool-incentives module account account
-	// poolIncentivesCoins := sdk.NewCoins(k.GetProportions(ctx, mintedCoin, proportions.PoolIncentives))
-	// err = k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, poolincentivestypes.ModuleName, poolIncentivesCoins)
-	// if err != nil {
-	// 	return err
-	// }
-
-	devRewardCoin := k.GetProportions(ctx, mintedCoin, proportions.DeveloperRewards)
-	devRewardCoins := sdk.NewCoins(devRewardCoin)
-	// This is supposed to come from the developer vesting module address, not the mint module address
-	// we over-allocated to the mint module address earlier though, so we burn it right here.
-	err = k.bankKeeper.BurnCoins(ctx, types.ModuleName, devRewardCoins)
+	// Allocate usage incentives to incentives module account
+	usageIncentives := sdk.NewCoins(k.GetProportions(ctx, mintedCoin, proportions.UsageIncentives))
+	err = k.bankKeeper.SendCoinsFromModuleToModule(
+		ctx,
+		types.ModuleName,
+		incentivestypes.ModuleName,
+		usageIncentives,
+	)
 	if err != nil {
 		return err
 	}
-	if len(params.WeightedDeveloperRewardsReceivers) == 0 {
-		// fund community pool when rewards address is empty
-		err = k.distrKeeper.FundCommunityPool(ctx, devRewardCoins, k.accountKeeper.GetModuleAddress(types.DeveloperVestingModuleAcctName))
-		if err != nil {
+
+	// Allocate community pool inflation (remaining module balance) to community
+	// pool address
+	moduleAddr := sdk.AccAddress(types.ModuleAddress.Bytes())
+	communityPool := sdk.NewCoins(k.bankKeeper.GetBalance(ctx, moduleAddr, mintedCoin.Denom))
```

Comment:
nit, rename to `communityPoolAmt`

Reply:
also, you can get transfer the entire remaining balance using `GetBalances(ctx)`